docs/src/main/paradox/user/lang/java/effects.md

```diff
@@ -14,12 +14,12 @@ This could be looked up in the constructor of the entity, for later use, so it d
 
 ## Forwarding command
 
-To forward a command, @javadoc[`ClientActionContext.forward()`](io.cloudstate.javasupport.ClientActionContext#forward-io.cloudstate.javasupport.ServiceCall-) may be invoked. The `CommandContext` for each entity type implements `ClientActionContext` and so offers this function. If the hot items effect service call above were to be invoked as a forward, it would might look like this:
+The `CommandContext` for each entity type implements `ClientActionContext` to allow forwarding a command by invoking @javadoc[`ClientActionContext.forward()`](io.cloudstate.javasupport.ClientActionContext#forward-io.cloudstate.javasupport.ServiceCall-). For example, the `item` being processed in the `addItem` command, meets a condition to be a "hot" item. We can make the `Hotitems` entity aware of that item by forwarding a command:
 
 @@snip [ShoppingCartEntity.java](/docs/src/test/java/docs/user/effects/ShoppingCartEntity.java) { #forward }
 
 ## Emitting an effect
 
-To emit an effect, @javadoc[`EffectContext.effect()`](io.cloudstate.javasupport.EffectContext#effect-io.cloudstate.javasupport.ServiceCall-boolean-) may be invoked. The `CommandContext` for each entity type implements `EffectContext` and so offers this function. If the hot items effect service call above were to be invoked as an effect, it would might look like this:
+The `CommandContext` for each entity type implements `EffectContext` to allow emitting an effect by invoking @javadoc[`EffectContext.effect()`](io.cloudstate.javasupport.EffectContext#effect-io.cloudstate.javasupport.ServiceCall-boolean-). Similarly to the example above, instead of forwarding a command, we want the `Hotitems` entity to emit an effect. We would invoke the effectful service call as:
```

Comment:
This comment is almost identical to that of `ClientActionContext.forward()`. Would the reader be confused when to forward a command and when to emit en effect?

Reply:
Good point. Maybe we should either have a short description of what the difference is, or another link to the general forwarding and effects page (which is also at the top of this page), or both. I think this page assumes that you've read that first.

Reply:
I'd rather reword, remind the difference and give the link to the general page. The reader is likely new to Cloudstate, would probably less familiar than us about the documentation structure.

Reply:
Sounds good 👍

Reply:
Attempted to reword + add a reminder link to common section. Kind of redundant reminder but could be helpful for devs new to the concept. Please review again and in case you suggest any change, your version will be final. Thanks